src/App.css

```diff
@@ -1,3 +1,13 @@
+@import url("https://fonts.googleapis.com/css2?family=Livvic:ital,wght@0,600;0,700;1,600;1,700&display=swap");
```

Comment:
To think about: when we merge both this PR and PR #1 into the main branch, where do we want the font import to live? We should only do it once.

Reply:
lets keep it in app.css
that will make it global